### PR TITLE
Update vision-of-the-thousand-brains-project.md

### PR DESCRIPTION
**Pull request by RichMorin**
minor tweaks

### CODE REVIEW

**Comment by tristanls** [comment]
Hi @RichMorin, I see a signed CLA with `Rich_Morin` GitHub username. Please use `RichMorin` to pass the CLA check.

**Comment by RichMorin** [comment]
I'm afraid I don't understand what you are asking me to do.  More generally, is there a conflict between the names RichMorin and Rich_Morin?  ELI5...

-r

**Comment by tristanls** [comment]
Thank you for signing the CLA!

@codeallthethingz, I'm tagging you to review these changes to high-level goal language. FYI @vkakerbeck.

**Comment by codeallthethingz** [comment]
I'm going to swap in @vkakerbeck as the reviewer for this page as it is the vision page that I transcribed from a paper she authored.

File: docs/overview/vision-of-the-thousand-brains-project.md [hunk 1]
```diff
@@ -9,16 +9,18 @@ We call the implementation described herein "Monty", in reference to Vernon Moun
...
+There are numerous advantages to sensorimotor learning and reference frames. At a high level, you can think about all the ways humans are different from today's AI. We learn quickly and continuously, constantly updating our knowledge of the world as we go about our day. We do not have to undergo a lengthy and expensive training phase to learn something new. We interact with the world and manipulate tools and objects in sophisticated ways that leverage our knowledge of how things are structured. For example, we can explore a new app on our phone and quickly figure out what it does and how it works based on other apps we know. We actively test hypotheses to fill in the gaps in our knowledge. We also learn from multiple sensors and our different sensors work together seamlessly. For example, we may learn what a new tool looks like with a few glances and then immediately know how to grab and interact with the object via touch. Finally, the basis for decisions is considerably less opaque than that found in Large Language Models, etc.
```
Comment (vkakerbeck) [inline docs/overview/vision-of-the-thousand-brains-project.md:22]: I wouldn't end the last sentence of the paragraph with ",etc." How about something like "Overall, our decisions and interactions with the world are based on structured models that we can leverage and combine in novel way to adapt to the wide range of tasks and circumstances we face every day."
We could add something about LLMs being just general function approximations, learning the statistical regularities in a dataset but I think that would go into too much depth for a vision page.

File: docs/overview/vision-of-the-thousand-brains-project.md [hunk 2]
```diff
@@ -9,16 +9,18 @@ We call the implementation described herein "Monty", in reference to Vernon Moun
...
\ No newline at end of file
+One of the most important discoveries about the brain is that most of what we think of as intelligence, from seeing, to touching, to hearing, to conceptual thinking, to language, is created by a common neural algorithm. All aspects of intelligence are created by the same sensorimotor mechanism. In the neocortex, this mechanism is implemented in each of the thousands of cortical columns. This means we can create many different types of intelligent systems using a set of common building blocks. The architecture we are creating is built on this premise. Monty will provide the core components and developers will then be able to assemble widely varying AI and robotics applications using these components in different numbers and arrangements. Any engineer will be able to create AI applications using the Platform without requiring huge computational resources or background knowledge.
```
Comment (vkakerbeck) [inline docs/overview/vision-of-the-thousand-brains-project.md:26]: I don't see any difference here and also diff checker doesn't show anything. Am I missing something? If not, can you remove this "change" from the PR?

**Review by vkakerbeck — COMMENTED** [review]
Thanks for the updates! Its a bit hard to see the diffs here so I used https://www.diffchecker.com/text-compare/ but even with that the last paragraph didn't show anything. Lmk if I am missing something.

File: docs/overview/vision-of-the-thousand-brains-project.md (hunk 2 above)
Comment (tristanls) [reply docs/overview/vision-of-the-thousand-brains-project.md:26]: The last paragraph adds a new line to the end of the file. These don't show up in GitHub.

Screenshot 2024-12-12 at 09 36 02

Comment (vkakerbeck) [reply docs/overview/vision-of-the-thousand-brains-project.md:26]: Ahh I see :D You would think github could show this as a bit of a smaller diff than the entire paragraph...
